app/main.py

```diff
@@ -2,5 +2,14 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_memory = {}
+
+    def inner(*args) -> int:
+        key = tuple([*args])
```

Comment:
Tuple can already take iterable arguments, so we don't need to use [] first for that. Use only tuple